R/rvar-summaries-within-draws.R

```diff
@@ -0,0 +1,160 @@
+# Summaries over array elements, within draws ---------------------------------------------------------
+
+#' Summaries of random variables over array elements, within draws
+#'
+#' Compute summaries of random variables over array elements and within draws,
+#' producing a new random variable of length 1 (except in the case of
+#' `rvar_range()`, see 'Details').
+#'
+#' @param ... [`rvar`]s
+#' @template args-rvar-summaries-na.rm
+#' @param constant For `rvar_mad()`, a scale factor for computing the median
```

Comment:
refer to `stats::mad` for justification of this constant default?

Reply:
good idea